Pagination layout. Missing icons and weird display

bertmert opened the pull request:
Test:
- Rename /administrator/templates/isis/html/pagination.php
  to
  paginationxxxxxxxxxxxxxx.php
- Thus, Isis uses pagination from
  layouts/joomla/pagination/link.php
- Now, see backend list views, e.g. article manager:

Pagination in Joomla staging

![pagination-staging](https://cloud.githubusercontent.com/assets/11038612/6958609/adb63250-d90e-11e4-9391-1caff383dfd9.jpg)

Pagination in Joomla Staging after patch
![pagination-staging-after](https://cloud.githubusercontent.com/assets/11038612/6958626/f48e7a5c-d90e-11e4-8025-026f49db86d9.jpg)

bertmert committed:
Missing icons in pagination

infograf768 commented:
I can't reproduce this issue. Icons display here in staging. Their Css are present.
![screen shot 2015-04-02 at 08 08 42](https://cloud.githubusercontent.com/assets/869724/6958735/4c3480c0-d910-11e4-9a1b-924823e5afe8.png)

bertmert commented:
@infograf768 Yes, sorry. Please see added test instructions above.

infograf768 commented:
Works OK for what it wants to solve. 

Note: Using the layout gives weird display A supplementary Display dropdown + the pagination bar is not any more aligned left.

![screen shot 2015-04-02 at 09 22 18](https://cloud.githubusercontent.com/assets/869724/6959634/65364ad0-d91b-11e4-9e2d-72a221bf9443.png)

bertmert committed:
Remove inline-style center. showLimitBox false

- Alignment center should be set by CSS file if needed.
- Option showLimitBox by default false now because more often needed in Joomla.

Fedik commented:
thanks!
works good,
but I would not change `$showLimitBox   = $options->get('showLimitBox', false);`, because it could have effect on extension that already expect  different "default value"

bertmert commented:
> but I would not change $showLimitBox = $options->get('showLimitBox', false);, because it could have effect on extension that already expect different "default value"

@Fedik @infograf768 @all Please discuss ;-) I really don't know if we should ignore B/C here or not.

Fedik commented:
I do not see a reason to change this part of code.

bertmert committed:
Regression. showLimitBox back to true

bertmert commented:
@test I think all is done now and PR is ready to commit.

erikvandoorne commented:
I've tested this and the patch solves the issue. Please note the pagination becomes left-aligned after the patch.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6633">issues.joomla.org/joomla-cms/6633</a>.</sub>

zero-24 commented:
RTC based on various testing Thanks :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6633">issues.joomla.org/joomla-cms/6633</a>.</sub>

Bakual commented:
Doesn't this break each template which uses that (default) JLayout?

bertmert commented:
No core component or template is using this JLayout.
I'm using it in a FE template but use an override because of missing icons and hard coded center style.

Another alternative would be to add missing icon-classes to /media/jui/less/icomoon.less

Close PR and remove RTC please if you see B/C issues.

Bakual commented:
As I understand it, this is the default pagination layout used if no overrides are present? I haven't checked in detail but it's what I would expect.
It may be that core templates don't use this, but if there is a chance that 3rd party templates use it, then we can't simply change it and make those templates break.

You could maybe add additional classes to the output, having it contain two classes instead of replacing it.

bertmert commented:
Please remove RTC.

zero-24 commented:
@bertmert do you try the proposed fix by @Bakual ?

bertmert commented:
It's an issue of time that I don't have at the moment and the next weeks. I think PR is not that important and easy to reproduce if so. wants to take over..

zero-24 commented:
@bertmert see: https://github.com/joomla/joomla-cms/pull/7175 Thanks!
